src/errors.rs

```diff
@@ -4,7 +4,7 @@ use core::fmt::{Display, Formatter, Result};
 use derive_more::Error;
 
 #[cfg_attr(feature = "std", derive(Error))]
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Copy, Debug, PartialEq)]
```

Comment:
The `Copy` instance prevents us from adding a non-trivial object inside (i.e stacktraces).

Reply:
At the moment this error doesn't contain it.
Once you decide to add, you can remove Copy (adding a new variant is also considered to be a breaking change).